examples/snippets/clients/completion_client.py

```diff
@@ -0,0 +1,80 @@
+"""MCP client example showing completion usage.
+
+This example demonstrates how to use the completion feature in MCP clients.
+cd to the `examples/snippets` directory and run:
+    uv run completion-client
+"""
+
+import asyncio
+import os
+
+from mcp import ClientSession, StdioServerParameters
+from mcp.client.stdio import stdio_client
+from mcp.types import PromptReference, ResourceTemplateReference
+
+# Create server parameters for stdio connection
+server_params = StdioServerParameters(
+    command="uv",  # Using uv to run the server
+    args=["run", "server", "completion", "stdio"],  # Server with completion support
+    env=os.environ.copy(),  # Pass environment variables
```

Comment:
This is dangerous, I think we should be more conscious about it.

Can you pass a single environment variable instead to show the point?